### ISSUE TITLE
Reason for using pool length 2?

### ISSUE THREAD

**Issue by shashankg7** [alexander-rakhlin/CNN-for-Sentence-Classification-in-Keras#8]
Hi,

I am not able to understand your motivation behind using pool length of only 2 in maxpooling layer.

In the paper, authors have used global max pooling, i.e. maximum activation from each feature map, one scalar from each feat. map. While you are taking half of the activations from each feature map

**Comment by anonoz** [comment]
@shashankg7 Interestingly, I tried to set the `pool_length = filter_size`, but I have gotten lower validation accuracy. Have you tried other combos?

**Comment by alexander-rakhlin** [comment]
I suggest you to refresh how maxpooling works. Unlike global pooling, maxpooling yields multiple values from a feature map, global pooling - just a single value.
